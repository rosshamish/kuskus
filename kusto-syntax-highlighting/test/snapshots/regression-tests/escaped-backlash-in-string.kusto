let scannerId = "9d632724-5c18-43e9-9314-555e9587b30c";
cluster("foo").database("bar").table("baz")
| extend FileNameParts = split(FullFileName, "Modified\\")
| extend FileName = trim("'", FileName)