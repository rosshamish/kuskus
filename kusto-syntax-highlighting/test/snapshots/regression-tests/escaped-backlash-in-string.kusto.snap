>let scannerId = "9d632724-5c18-43e9-9314-555e9587b30c";
#^^^ source.kusto keyword.control.kusto
#   ^^^^^^^^^^^^^ source.kusto
#                ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kusto string.quoted.double.kusto
#                                                     ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                                                      ^^ source.kusto
>cluster("foo").database("bar").table("baz")
#^^^^^^^ source.kusto meta.special.database.kusto support.function.kusto
#       ^ source.kusto meta.special.database.kusto
#        ^ source.kusto meta.special.database.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#         ^^^ source.kusto meta.special.database.kusto string.quoted.double.kusto
#            ^ source.kusto meta.special.database.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#             ^ source.kusto meta.special.database.kusto
#              ^ source.kusto
#               ^^^^^^^^ source.kusto meta.special.database.kusto support.function.kusto
#                       ^ source.kusto meta.special.database.kusto
#                        ^ source.kusto meta.special.database.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                         ^^^ source.kusto meta.special.database.kusto string.quoted.double.kusto
#                            ^ source.kusto meta.special.database.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                             ^ source.kusto meta.special.database.kusto
#                              ^ source.kusto
#                               ^^^^^ source.kusto support.function.kusto
#                                    ^ source.kusto
#                                     ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                                      ^^^ source.kusto string.quoted.double.kusto
#                                         ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                                          ^^ source.kusto
>| extend FileNameParts = split(FullFileName, "Modified\\")
#^^ source.kusto
#  ^^^^^^ source.kusto keyword.other.query.kusto
#        ^^^^^^^^^^^^^^^^^ source.kusto
#                         ^^^^^ source.kusto support.function.kusto
#                              ^^^^^^^^^^^^^^^ source.kusto
#                                             ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                                              ^^^^^^^^ source.kusto string.quoted.double.kusto
#                                                      ^^ source.kusto string.quoted.double.kusto constant.character.escape.kusto
#                                                        ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                                                         ^^ source.kusto
>| extend FileName = trim("'", FileName)
#^^ source.kusto
#  ^^^^^^ source.kusto keyword.other.query.kusto
#        ^^^^^^^^^^^^ source.kusto
#                    ^^^^ source.kusto support.function.kusto
#                        ^ source.kusto
#                         ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                          ^ source.kusto string.quoted.double.kusto
#                           ^ source.kusto string.quoted.double.kusto punctuation.definition.string.kusto
#                            ^^^^^^^^^^^^ source.kusto